Finjector.Web/ClientApp/src/shared/Header.tsx

```diff
@@ -0,0 +1,19 @@
+const Header = () => (
+  <div className="header">
+    <div className="container">
+      <div className="row justify-content-between">
+        <div className="col">
+          <h1>UC DAVIS CoA Picker</h1>
+        </div>
+        <div className="col text-end">
+          <p>
+            <a className="profile-link" href="#">
```

Comment:
I'd say just drop this part

Reply:
agreed, can leave name if you want but drop link and no signout